src/backend/joanie/core/api/admin.py

```diff
@@ -36,7 +41,7 @@ class ProductViewSet(viewsets.ModelViewSet):
     Admin Product ViewSet
     """
 
-    authentication_classes = [authentication.SessionAuthentication]
+    authentication_classes = [SessionAuthenticationWithAuthenticateHeader]
     permission_classes = [permissions.IsAdminUser & permissions.DjangoModelPermissions]
```

Comment:
why don't you need to add permissions.IsAuthenticated here as you did above?

Reply:
Oh! Above this a test residue...